docs/blog.md

```diff
@@ -1,5 +1,12 @@
 # Microsoft MakeCode Blog
 
+## [MakeCode for LEGO MINDSTORMS](/blog/lego/05-15-2018)
```

Comment:
should we have @boardname@ to get the full (r) name?

Reply:
Alas, this would result in "MakeCode for unknown macro". Need a product macro map for several boards maybe.